### ISSUE TITLE
Overlapping nodes?

### ISSUE THREAD

**Issue by iLShado** [adrienverge/familytreemaker#3]
It seems this code has a maximum size for a given generation, for instance I wanted to put my grandmother's five children but on the 4th one, where there's supposed to be another son,  I can only see a line from the 3rd children that connects itself with the parenting line, and there's no node for that 4th person, but the weirdest thing is that after that 4th person, the 5th one fits just perfectly, so I don't know how and why is this occurring, because I followed the syntaxis instructions and all the other generations that have fewer members work, so I guess it has a limited amount of people per generation?

![error](https://user-images.githubusercontent.com/48779732/130834026-b6f479be-26e3-4e33-98f2-47c73df45399.png)

I must note that the same error occurs with the partner of a given person from that generation, as it only shows the wife and then a self connecting line (and the line that goes to their children) and right next to that node, her sister.
![error2](https://user-images.githubusercontent.com/48779732/130834832-d10813c5-543d-4b47-968a-d4370275f904.png)

**Comment by adrienverge** [comment]
Hello,

This is interesting, I wasn't aware. It's probably due to the way GraphViz handles DOT files with this specific configuration of nodes.

For info, this code was written 8 years ago and the project is unmaintained.

**Comment by AreYouLoco** [comment]
I also had some artifacts using this script. But just be sure to have unique ID's when using the same name and regenerate your tree.

**Comment by xeruf** [comment]
Yep also having issues beyond three generations, which is very unfortunate as I otherwise appreciate this project highly
